core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -2233,67 +2236,67 @@ object Parser extends ParserInstances {
 }
 
 abstract class ParserInstances {
```

Comment:
since there are two companions now, is there any reason not to just make this `object Parser0`?

Also, should we alias all the def foo0 methods in here so people can do P0.string or P.string rather than P.string0 which is a bit weird.

Reply:
For the first, I'll try it out. 

For the second, I don't have a strong opinion either way. Having a `P0.foo` as an alias for `P.foo0` meaning something subtly but breakingly different from `P0.foo` is a bit of a foot gun. I'm not sure if the added convenience is worth it.

Reply:
I don't follow how they would be different. Can you explain? I am proposing they point to exactly the same code.

Reply:
I meant to say the difference between P.foo and P0.foo being the foot gun, but in style I mistyped it.